catalog/monitors/rds.yaml

```
@@ -0,0 +1,179 @@
+# The official Datadog API documentation with available query parameters & alert types:
+# https://docs.datadoghq.com/api/v1/monitors/#create-a-monitor
+
+rds-cpuutilization:
+  name: "(RDS) CPU Utilization above 90%"
+  type: metric alert
+  query: |
+    avg(last_15m):avg:aws.rds.cpuutilization{*} by {dbinstanceidentifier} > 90
+  message: |
+    {{#is_warning}}
+    ({dbinstanceidentifier}) CPU Utilization above 85%
+    {{/is_warning}}
+    {{#is_alert}}
+    ({dbinstanceidentifier}) CPU Utilization above 90%
+    {{/is_alert}}
+  escalation_message: ""
+  tags: [ "ManagedBy:Terraform" ]
+  notify_no_data: false
+  notify_audit: true
+  require_full_window: false
```

```suggestion
  require_full_window: true
```

```
require_full_window a Boolean indicating whether this monitor needs a full window of data before it’s evaluated. We highly recommend you set this to False for sparse metrics, otherwise some evaluations are skipped. Default: True for “on average”, “at all times” and “in total” aggregation. False otherwise.
```
https://docs.datadoghq.com/monitors/guide/monitor_api_options/

Please check other options (if they are correctly set according to the collected metrics)